src/Pure.Blazor.Components/Forms/PureInput.razor.cs

```diff
@@ -113,6 +94,10 @@ public partial class PureInput
     [Parameter] public EventCallback<MouseEventArgs> OnSuffixClick { get; set; }
     [Parameter] public EventCallback<FocusEventArgs> OnBlur { get; set; }
 
+    [Parameter] public EventCallback<KeyboardEventArgs> OnKeyPress { get; set; }
```

Comment:
if we're enabling these for the purposes of detecting 'enter', maybe we should have an `OnEnter` event instead?

Reply:
I've added an OnEnter event, since it would be useful to avoid repeating that

I can get rid of the OnKeyX events, if you think it just adds bloat.

Reply:
I'd remove it since we won't be actively using it and it increases overhead (probably little, but still)